test/test_decomp.py

```diff
@@ -742,6 +742,9 @@ def all_aten_overloads():
         # This is for operators that are only registered in some CI
         # configurations, so would cause the test to fail
         allow_list = {aten.get_gradients.default}
+        # Skip check for _int_mm
```

Comment:
@ngimel - Is this what you had in mind in comment https://github.com/pytorch/pytorch/pull/95681#issuecomment-1447533599 ?

Reply:
This test asserts expected outcome on line 752, you should modify expected file (`HasDecompTest.test_has_decomposition.expect`) to include `_int_mm` (you can do it passing `--accept` arg to the test, or manually).